package.json

```diff
@@ -7,7 +7,8 @@
 	"type": "commonjs",
 	"exports": {
 		"import": "./dist/index.mjs",
-		"require": "./dist/index.js"
+		"require": "./dist/index.js",
+		"types": "./dist/index.d.ts"
```

Comment:
I think this needs to come first: https://github.com/eps1lon/dom-accessibility-api/pull/996